packages/manager/modules/veeam-enterprise/src/dashboard/controller.js

```diff
@@ -73,6 +67,10 @@ export default class VeeamEnterpriseDashboardCtrl {
     };
   }
 
+  onManageClick() {
```

Comment:
related to this :
https://github.com/ovh/manager/pull/6765/files#r841654120

Reply:
I applied.